doc/CONFIGURATION.md

```diff
@@ -141,3 +149,453 @@ node:
 # the file we will dump logs into
 log_file: ./logger.log
 ```
+
+# Datastore Configuration
+
+The `datastore` section(s) of the yaml config file is used to config key-value datastores. These key-value datastores are used throughout the entire libp2p, and ipfs stack. Every `datastore` section shares the same configuration options with each other. In fact in the code-base, they all use the same code.
```

Comment:
Remove "In fact in the code-base, they all use the same code."

Reply:
ok